resources/dicts/thoughts/dead/starclan/newborn.json

```diff
@@ -27,5 +33,20 @@
         "random_living_status": [
             "living"
         ]
+    },
+    {
+        "id": "gen_dead_newborn_to_dead_other",
+        "thoughts":[
+            "Is being watched by r_c",
+            "r_c is bathing them",
```

Comment:
"is being bathed by r_c" fits the thought format better

Reply:
sorry! i copied it over from the ideas doc and didn't proofread, my bad